### DISCUSSION TITLE
Quarto overwrite and then deletes a notebook file with the same base name as the quarto file

### DISCUSSION THREAD

**Discussion by vnijs** [quarto-dev/quarto-cli#11381] [Q&A]
### Description

Suppose I have a file my-file.ipynb and my-file.qmd in the same directory. I want to keep and use both files. If I now render the qmd file it seem to (1) overwrite my-file.ipynb and then (2) delete my-file.ipynb. That seems a bit destructive. I can change the names of the files but I'd like to know if there is a way to stop quarto from overwriting and deleting files in this way. Thanks

**Comment by mcanouil** [comment]
This issue has already been extensively discussed in several places. Please take a look at those discussions for more information.

- https://github.com/quarto-dev/quarto-cli/issues/10841
- https://github.com/quarto-dev/quarto-cli/issues/8727
- https://github.com/quarto-dev/quarto-cli/issues/4802

Which version of Quarto are you using? Note that I cannot reproduce the behaviour you described on latest Quarto CLI version.
